src/Compilers/CSharp/Test/Syntax/Parsing/TopLevelStatementsParsingTests.cs

```diff
@@ -17,11 +17,6 @@ public sealed class TopLevelStatementsParsingTests : ParsingTests
     {
         public TopLevelStatementsParsingTests(ITestOutputHelper output) : base(output) { }
 
-        protected override SyntaxTree ParseTree(string text, CSharpParseOptions options)
```

Comment:
Could you please clarify what is affected by removal of this method? For example, are any tests in this class changed from failing to passing?

Reply:
> Could you please clarify what is affected by removal of this method? For example, are any tests in this class changed from failing to passing?

CSharp9 is default now, so no need to override the default. Also, we don't want the test to be tied to CSharp9 forever. There was no change in behavior because of this change.